coalib/misc/CheckVersion.py

```diff
@@ -0,0 +1,16 @@
+from pkg_resources import get_distribution, DistributionNotFound
+
+
+def check_version(package_name, version):
+    """
+    Check if a particular package version is present or not.
+
+    :param package_name: Package name.
+    :param version:      Version number to match.
+    :return:             ``True`` if package is with given ``version``,
```

Comment:
Btw you can also use this format for docstring:
```
:param package_name:
    Package name.
:param version:
    ...
...
```
As this is what we try to switch to (once the DocumentationBear works we can enforce the style)

Reply:
Okay done :)